hack/local-up-cluster.sh

```diff
@@ -362,13 +390,24 @@ function start_apiserver {
     # Ensure CERT_DIR is created for auto-generated crt/key and kubeconfig
     sudo mkdir -p "${CERT_DIR}"
 
+    # Create client ca
+    sudo /bin/bash -e <<EOF
+    rm -f "${CERT_DIR}/client-ca.crt" "${CERT_DIR}/client-ca.key"
+    openssl req -x509 -sha256 -new -nodes -days 365 -newkey rsa:2048 -keyout "${CERT_DIR}/client-ca.key" -out "${CERT_DIR}/client-ca.crt" -subj "/C=xx/ST=x/L=x/O=x/OU=x/CN=ca/emailAddress=x/"
+    echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","client auth"]}}}' > "${CERT_DIR}/client-ca-config.json"
+EOF
+    create_client_certkey kubelet
+    create_client_certkey kube-proxy
+    create_client_certkey controller
```

Comment:
This is a controller-bootstrapper identity.  We expect controllers to run as service accounts and only bootstrapping will require special powers.  It should have a `system:masters` group.